### DISCUSSION TITLE
ApexChart-card graph does not show on Ipad.

### DISCUSSION THREAD

**Discussion by fmcoolen** [RomRider/apexcharts-card#797] [Q&A]
I installed apexcharts-card on HA. Created a graph showing electricity prices. Graph shows fine on Windows  and  Iphone (HA app), but not on iPad. Tried two iPad's. Already cleared cache from browsers (Safari and Chrome). Only the Y-as with prices is visible.

**Comment by Stecia** [comment]
I have the same problem, I've used PC, iPhone 14 pro/IOS 18.5, iPad 7th gen/IOS 18.1.1/ (2019) and they all are working fine, except my iPad air2/IOS 15.8.4/ (2015) and specially this device I want to use as a always-on display. Used the graph for the dynamic electricity prices.

Except for the PC, I use the HomeAssistant app, also on the iPad air2, but if I try the browser it still does not work.

Anyone a suggestion??

![image](https://github.com/user-attachments/assets/e22b8453-ac50-47ef-838f-78a93c2b85d9)

**Comment by RomRider** [comment]
That is probably related to a too old version of the the engine used in your air2 ipad (IOS 15 is 5 years old)
Maybe the latest apexcharts-card just released fixes that, but I have no way to test.

**Comment by vingerha** [comment]
On my airpad pro v2021 with updated iOS ... no issues

**Comment by Saentist** [comment]
Apple problem with JVM to small size 128kb
